src/server/system_services/bucket_server.js

```diff
@@ -606,6 +607,10 @@ function get_bucket_changes(req, update_request, bucket, tiering_policy) {
         get_bucket_changes_versioning(req, bucket, update_request, single_bucket_update, changes);
     }
 
+    if (!_.isUndefined(update_request.force_md5_etag)) {
```

Comment:
I don't see why we need to be explicit about undefined, we can just check with regular if
```suggestion
    if (update_request.force_md5_etag) {
```

Reply:
I think regular if will not work here, as `force_md5_etag` is a boolean flag and if the value of `update_request.force_md5_etag` is `false` then the condition is not met and value is not updated for `single_bucket_update.force_md5_etag`